### ISSUE TITLE
Install.sh doesn't clean up old docker images

### ISSUE THREAD

**Issue by cdnninja** [thorrak/fermentrack#754]
Older docker images are left without deletion.  This would fill the SD cards after multiple updates. 

![image](https://github.com/thorrak/fermentrack/assets/6373468/fa25715d-cd2d-45a8-b477-43ee5c470ba4)

**Comment by thorrak** [comment]
Yeah -- agreed. I should probably do a `docker image prune` at the end. 

Moving this to `fermentrack-tools` as that's where the install script lives to track against.
